CONFIGURATION.md

```diff
@@ -317,16 +320,22 @@ of the images in the pipeline.
 `size` is the required quantity of the volume to create in bytes. Use Kubernetes
 resource quantity semantics (e.g. `1Gi` or `500M`).
 
-# `tests.secret` 
+# `test.secrets`
+`Secrets` is an array of secrets. This  enables users to mount multiple secrets
+inside test container. It is users responsibility to make sure secret is available
+in the temporary namespace. This can be done by providing flag `--secret-dir`
+to ci-operator in prow configuration.
+
+# `tests.secret` Deprecated
```

Comment:
Since this is not being used anymore, the configuration will break either way. Lets remove it and open a PR to release repo as a follow-up to fix the ci-operator configurations.

Reply:
The config should not break, `secret` is internally turned into `secrets` with a single element, which seems fine to me. Here we can migrate to new config "properly" (merge this change, then migrate to new config, then remove the old code), there's no reason to do it via a breaking change.